### DISCUSSION TITLE
Многопоточность

### DISCUSSION THREAD

**Discussion by spqr911** [malikzh/NCANode#80] [General]
Добрый день!
Классно что такое решение сделали. 

Есть пара вопросов по работе приложения: 

- Ваш сервер может обрабатывать несколько запросов одновременно? 
- Поддерживает ли kalkan.jar многопоточность? 

Мы сейчас используем kalkan.COM и пробовали использовать библиотеку на C у нас возникли проблемы с многопоточностью..., библиотека при больших нагрузках либо начинает врать (при одних и тех же входных данных дает разные ответы) либо говорит что она не инициализирована, разные ошибки получаем в зависимости от подхода к использованию библиотеки.

**Comment by malikzh** [comment]
День добрый.

- Я использую этот проект в высоко-нагруженных проектах, пока проблем обнаружено не было. Имеется образ докер, поэтому без проблем можно будет масштабировать горизонтально.
- Не уверен точно. Здесь я могу порекомендовать обратиться в PKI. Если у меня есть вопросы по калкану, я пишу им на почту.

**Comment by spqr911** [comment]
Спасибо
________________________________
